test/fuzz/utility.h

```diff
@@ -99,6 +99,20 @@ inline T fromHeaders(
   return header_map;
 }
 
+// Convert from test proto Metadata to MetadataMap
+inline Http::MetadataMapVector fromMetadata(const test::fuzz::Metadata& metadata) {
+  Http::MetadataMapVector metadata_map_vector;
+  if (!metadata.metadata().empty()) {
+    Http::MetadataMap metadata_map;
+    for (const auto& pair : metadata.metadata()) {
+      metadata_map.insert(pair);
+    }
+    Http::MetadataMapPtr metadata_map_ptr = std::make_unique<Http::MetadataMap>(metadata_map);
```

Comment:
Nit: why not allocate this above the loop and then use the map in the loop? This avoids a copy.

Reply:
done